### PR TITLE
feat: Implement tap targets for butttons in chat activity

### LINKED ISSUE #2166: Implement tap targets for chat activity buttons
**Describe the feature you'd like**
Currently user is not able to judge the use of the floating action buttons in the chat activity.Therefore we can use tap targets for these buttons to help user.These tap targets will only be shown at first installation of the app. 

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by chanmol1999**
Fixes #2166

Changes:chat_activity,strings.xml
Screenshots for the change: 
video_20190413_150542

### CODE REVIEW

**Comment by chanmol1999** [comment]
Please review @iamareebjamal @batbrain7 @arundhati24

**Comment by chanmol1999** [comment]
Please review sir @iamareebjamal @batbrain7 
Do I need to make any changes.

**Comment by iamareebjamal** [comment]
Peer Review

**Comment by Bhavnaharitsa** [comment]
please add the translations in all the languages for the new strings you have added.

**Comment by Bhavnaharitsa** [comment]
But, do we need this feature in the first place? Because I don't think most of the android chat apps have this feature.
@chanmol1999 This was my opinion. If you feel this is a good feature, then you can keep it and let the mentors decide if we need this or not.

**Comment by Bhavnaharitsa** [comment]
@iamareebjamal I have reviewed
Thank you.

File: app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt [hunk 1]
```diff
@@ -82,7 +88,6 @@ class ChatActivity : AppCompatActivity(), IChatView {
             textToSpeech?.stop()
         }
     }
-
```
Comment (immadisairaj) [inline app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt]: Why is this line removed?

File: app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt [hunk 2]
```diff
@@ -111,6 +116,50 @@ class ChatActivity : AppCompatActivity(), IChatView {
         }
     }
 
+    @TargetApi(Build.VERSION_CODES.LOLLIPOP)
```
Comment (immadisairaj) [inline app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt]: So the minimum version needed to use this feature is Lollipop.

File: app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt [hunk 3]
```diff
@@ -111,6 +116,50 @@ class ChatActivity : AppCompatActivity(), IChatView {
...
+                                        .transparentTarget(false)
+                                        .icon(getDrawable(R.drawable.ic_open_search))// Specify whether the target is transparent (displays the content underneath) // Specify a custom drawable to draw as the target
+                                        .targetRadius(100)),
+                        TapTargetView.showFor(this, // `this` is an Activity
+                                TapTarget.forView(findViewById(R.id.fabsetting), getString(R.string.susi_icon_title), getString(R.string.susi_icon_description))
+                                        // All options below are optional
+                                        .outerCircleColor(R.color.colorPrimaryDark)
+                                        .outerCircleAlpha(0.96f)
+                                        .targetCircleColor(R.color.md_lime_50)
+                                        .titleTextSize(30)
```
Comment (immadisairaj) [inline app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt]: Don't use hard coded dimensions

File: app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt (hunk 2 above)
Comment (Bhavnaharitsa) [reply app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt]: but we don't use lollipop
